airflow_dags/wikipedia/wikiextractor/WikiExtractor.py

```diff
@@ -2974,6 +2974,8 @@ def process_dump(input_file, template_file, out_file, file_size, file_compress,
     # Mapper process
     page_num = 0
     for page_data in pages_from(input):
+        if page_num == 4000:
```

Comment:
Will remove this later, but this is where you can break at a specific numbers of articles